ml/v1/ml.proto

```diff
@@ -59,7 +59,40 @@ service MLService {
             body: "*"
         };
     }
+    
+    // List the available size configurations for MLServices.
+    // Note that the returned size specifications are applied for ML Jobs.
+    // Required permissions:
+    // - ml.mlservicessize.list on the deployment (if deployment_id is provided)
+    // - None, authenticated only
+    rpc ListMLServicesSizes(ListMLServicesSizesRequest) returns (MLServicesSizeList);
+}
+
+// List of MLServicesSize.
+message MLServicesSizeList {
+    // Items in this list.
+    repeated MLServicesSize items = 1;
+}
+
+message ListMLServicesSizesRequest {
+    // Optional ID of the Deployment for which sizes are being requested.
+    // If set, the response will exclude any sizes that are unavailable for the specified deployment model.
+    string deployment_id = 1;
+}
 
+// MLServicesSize represents the resources allocated for MLServices.
+// Note that the specified configuration is applied for the ML jobs.
+message MLServicesSize {
+    // Identifier of the size configuration.
+    string size_id = 1;
+    // If set, this is the default size when unspecified in MLServices.
+    bool is_default = 2;
+    // Amount of CPU allocated (in vCPU units)
+    float cpu = 3;
+    // Amount of Memory allocated (in GB)
+    int32 memory = 4;
+    // Amount of GPU Memory allocated (in GB)
+    int32 gpu_memory = 5;
```

Comment:
please change to gpu, see: notebook sizes for an sample at: https://github.com/arangodb-managed/apis/blob/efef24e5e79b84e699ea03e342d1e755a4f83eb2/notebook/v1/notebook.proto#L219